apps/nextjs/src/components/AppComponents/Chat/drop-down-section/modify-button.tsx

```diff
@@ -122,4 +125,23 @@ const ModifyButton = ({
   );
 };
 
+function handleLabelText({
+  text,
+  section,
+}: {
+  text: string;
+  section: string;
+}): string {
+  if (
+    section === "Misconceptions" ||
+    section === "Keyword learning points" ||
```

Comment:
I think this line should refer to the keywords section rather than the learning points.